src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -122,7 +122,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                     if (addresses.SingleOrDefault() != "http://localhost:5000")
                     {
                         var joined = string.Join(", ", addresses);
-                        throw new NotSupportedException($"Specifying address(es) '{joined}' is incompatible with also configuring endpoint(s) in UseKestrel.");
+                        _logger.LogWarning($"Address(es) '{joined}' were overridden. Binding to endpoints defined in UseKestrel instead.");
```

Comment:
I think the wording needs to be a little different. It may be confusing to someone specifying multiple addresses in the feature but only overriding a single one of those in `UseKestrel()`.

Maybe something like "Addresses in `UseUrls()`/`IServerAddressesFeature` [pick one] were [are? since it's something that will always happen when using both] overriden by the call to `UseKestrel()`. Binding to endpoints defined in `UseKestrel()` instead."